src/platform/site-wide/index.js

```diff
@@ -25,10 +24,6 @@ import '@department-of-veterans-affairs/formation/dist/formation';
  * @param {Store} commonStore The Redux store being used by this application
  */
 export default function startSitewideComponents(commonStore) {
-  if (document.querySelector('#vetnav-menu') !== null) {
-    addMenuListeners(document.querySelector('#vetnav-menu'), true);
-  }
```

Comment:
This is so confusing to me...we're using a React menu so I can't imagine we need an additional set of event listeners applied via `addMenuListeners`, but `#vetnav-menu'` _does_ exist in the DOM tree, so does that we mean _do_ need both (React component and listeners) or are we just needlessly applying this module by mistake?

CC: @department-of-veterans-affairs/frontend-review-group

Reply:
I think this is fine... The `addMenuListeners` doesn't add any listeners if the element passed in doesn't have `role="menu"` or `role="menubar"`, and it doesn't seem like `#vetnav-menu` has either.